src/engine/Scene.ts

```diff
@@ -122,6 +120,34 @@ module ex {
          // will be overridden
          this._logger.debug('Scene.onDeactivate', this);
       }
+
+      /**
+       * Initializes actors in the scene
+       */
+      public initializeChildren(): void {
```

Comment:
As we discussed, probably do `public _initializeChildren()` and mark as `@internal` for good measure. Hopefully that is enough warning to people.